hkm/static/hkm/js/cookie-consent.js

```diff
@@ -0,0 +1,46 @@
+(function($) {
+    var cookieName = "hki_consent"
+    var cookie = $.cookie(cookieName)
+    var _paq = window._paq = window._paq || [];
+
+    if(!cookie && _paq) {
+        _paq.push(['requireCookieConsent']);
+
+        $("#cookie-consent").fadeIn();
+        $( "#cookie-consent" ).animate({ "bottom": "0" }, "slow" );
```

Comment:
Using Jquery with a Sizzle selector `$(selector)` actually travels through a DOM-tree and creates a new object, so repeating it is not a good practise. If repeated and also for cleaner solution and to list the actors/roles, these should be collected to variables in start of the script. For example:

```javascript
 const $cookieConsentContainer = $("#cookie-consent"),
    $readMoreBtn = $("#cookie-consent__readmore-btn" );
// ...
```

Note that the jQuery objects usually are prefixewd with a dollar sign.

Reply:
More about that here: https://docs.tizen.org/application/web/guides/w3c/perf-opt/jquery-performance-improvement/#using-selectors

Reply:
ofcourse when using an id-selector, this does not optimize much, but it gives a cleaner structure... and like said, lists the actors.

Reply:
fixed